src/components/data-grid/types.ts

```diff
@@ -242,6 +242,10 @@ export interface DataGridProps {
   debouncedSearch?: boolean;
   settings?: DataGridSetting[];
   hideColumnToggle?: boolean;
+  isPagination?: boolean;
```

Comment:
```suggestion
  showPagination?: boolean;
```
I think sounds a little bit better